src/controllers/reactWebviewBaseController.ts

```diff
@@ -47,36 +52,68 @@ export abstract class ReactWebviewBaseController<State, Reducers>
     >;
     private _isFirstLoad: boolean = true;
     private _loadStartTime: number = Date.now();
+    private _endLoadActivity = startActivity(
+        TelemetryViews.WebviewController,
+        TelemetryActions.Load,
+    );
     private _onDisposed: vscode.EventEmitter<void> =
         new vscode.EventEmitter<void>();
     public readonly onDisposed: vscode.Event<void> = this._onDisposed.event;
     protected _webviewMessageHandler = async (message) => {
         if (message.type === "request") {
+            const endActivity = startActivity(
+                TelemetryViews.WebviewController,
+                TelemetryActions.WebviewRequest,
+            );
             const handler = this._webviewRequestHandlers[message.method];
             if (handler) {
-                const startTime = Date.now();
-                const result = await handler(message.params);
-                this.postMessage({ type: "response", id: message.id, result });
-                const endTime = Date.now();
-                sendActionEvent(
-                    TelemetryViews.WebviewController,
-                    TelemetryActions.WebviewRequest,
-                    {
+                try {
+                    const result = await handler(message.params);
+                    this.postMessage({
+                        type: "response",
```

Comment:
Are there other types in addition to `"response"`?

Reply:
For now, there isn't any other type. I have been meaning to add stronger types but haven't done that yet.